includes/class-wc-subscriptions-checkout.php

```diff
@@ -24,7 +24,7 @@ public static function init() {
 		add_action( 'woocommerce_checkout_order_processed', array( __CLASS__, 'process_checkout' ), 100, 2 );
 
 		// Same as above, but this is for the Checkout block.
-		add_action( '__experimental_woocommerce_blocks_checkout_order_processed', array( __CLASS__, 'process_checkout' ), 100, 1 );
+		add_action( 'woocommerce_blocks_checkout_order_processed', array( __CLASS__, 'process_checkout' ), 100, 1 );
```

Comment:
@opr to maintain backwards compatibility, what are your thoughts on surrounding these with version checks (see suggestion below)?

```suggestion
		if ( class_exists( 'Automattic\WooCommerce\Blocks\Package' ) && version_compare( \Automattic\WooCommerce\Blocks\Package::get_version(), '6.3.0', '>=' ) ) {
			add_action( 'woocommerce_blocks_checkout_order_processed', array( __CLASS__, 'process_checkout' ), 100, 1 );
		} else {
			add_action( '__experimental_woocommerce_blocks_checkout_order_processed', array( __CLASS__, 'process_checkout' ), 100, 1 );
		}
```

Reply:
Thanks for that suggestion, seems super obvious now 🤦🏼  I also wrapped it in a `is_experimental_build` so we don't call the experimental hooks if we're in a dev version of the plugin. It seems `version_compare` doesn't like the `-dev` suffix we use.